crates/types/src/lib.rs

```diff
@@ -172,7 +170,7 @@ pub struct HotShotConfig<KEY: SignatureKey, ELECTIONCONFIG: ElectionConfig> {
     /// List of known node's public keys and stake value for certificate aggregation, serving as public parameter
     pub known_nodes_with_stake: Vec<PeerConfig<KEY>>,
     /// All public keys known to be DA nodes
-    pub known_da_nodes: HashSet<PeerConfig<KEY>>,
+    pub known_da_nodes: Vec<PeerConfig<KEY>>,
```

Comment:
Why this change?

Reply:
Same as above, the ordering of the peers broke the validation because they're pulled in as the stake table